lib/auth-middleware/auth_middleware/__init__.py

```diff
@@ -204,9 +239,27 @@ def verify_userauth(id_token, group_name=None, internal_token_only=False):
         if not kid:
             logger.warning("kid not found. Aborting.")
             raise OAuthProblem('Invalid token')
+
+        # tokenClientID = unverified_payload.get('clientID')
+        # if not tokenClientID:
+        #     logger.warning("clientID not found. Aborting.")
+        #     raise OAuthProblem('Invalid token')
+        # clientID = request.view_args.get('clientID')
+        # if tokenClientID != clientID:
+        #     logger.warning("clientID does not match. Aborting.")
+        #     raise OAuthProblem('Invalid token to access clientID')
+
         valid_issuer = False
         keyset = None
         target_client_ids = None
+        checkAud = True
+
+        AUTH_PUBKEYS = os.getenv('AUTH_PUBKEYS', '').strip()
+        AUTH_ISSUER = os.getenv(
+            'AUTH_ISSUER', '').strip()
+        print("AUTH ISSUER: ", AUTH_ISSUER)
```

Comment:
Please either remove print statement or replace it with logger info or debug statement. Thanks.